examples/metro_graph_signal.py

```diff
@@ -30,41 +30,37 @@
 # %%
 import os
 import sys
+
+import matplotlib.pyplot as plt
 import pandas as pd
 from unidecode import unidecode
-import matplotlib.pyplot as plt
 
-from pygsp2.utils_examples import (
-    make_metro_graph, 
-    metro_database_preprocessing, 
-    plot_signal_in_graph,
-    fetch_data
-)
+from pygsp2.utils_examples import (fetch_data, make_metro_graph,
+                                   metro_database_preprocessing, plot_signal_in_graph)
 
 current_dir = os.getcwd()
 os.chdir(current_dir)
 
 # %% Load data
 assets_dir = os.path.join(current_dir, 'data')
-fetch_data(assets_dir, "metro")
+fetch_data(assets_dir, 'metro')
```

Comment:
Some changes go from double quotes to single quotes, and other changes are the other way around. Can we leave everything with single quotes?